src/ResourceManager/Compute/Commands.Compute/Extension/CustomScript/SetAzureVMCustomScriptExtensionCommand.cs

```diff
@@ -181,6 +181,13 @@ public class SetAzureVMCustomScriptExtensionCommand : VirtualMachineExtensionBas
             HelpMessage = "Set command to execute in private config.")]
         public SwitchParameter SecureExecution { get; set; }
 
+        [Parameter(
```

Comment:
Does it make sense to add this to the base cmdlet? Frankly, having > 3 or 4 positional parameters is unrealistic in any case.